modules/pel/peltool/peltool.py

```diff
@@ -571,6 +572,8 @@ def main():
                         dest='IDToDelete', help='Delete a PEL based on its ID')
     parser.add_argument('-D', '--delete-all', dest='deleteAll',
                         action='store_true', help='Delete all PELs')
+    parser.add_argument('--archive', dest='archive',
```

Comment:
I think we only need to add this argument if on a BMC system, as if not on the BMC you can just use -p to pass in the path to the archive dir.

Reply:
Comments address in the latest commit.